src/main/java/uk/gov/hmcts/reform/prl/controllers/EditAndApproveDraftOrderController.java

```diff
@@ -226,6 +240,17 @@ public AboutToStartOrSubmitCallbackResponse populateCommonFields(
                 CaseData.class
             );
             Map<String, Object> response = draftAnOrderService.populateCommonDraftOrderFields(authorisation, caseData);
+            CreateSelectOrderOptionsEnum selectedOrder = (CreateSelectOrderOptionsEnum) response.get("orderType");
+            List<String> errorList = new ArrayList<>();
```

Comment:
ErrorList can be created once beginning of the method and can be used wherever needed.